about/about-serverless.adoc

```diff
@@ -24,12 +23,77 @@ For additional information about the {ServerlessProductName} life cycle and supp
 ====
 
 // Add something about CLI tools
+[id="about-serverless-components_{context}"]
+== Components of {ServerlessProductName} 
+
+[id="about-serverless-knative-serving_{context}"]
+=== Knative Serving
+
+Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers. Serving simplifies the application deployment, dynamically scales based on in incoming traffic and supports custom rollout strategies with traffic splitting. 
+
+Other features of Knative Serving include:
```

Comment:
The important part from the IBM Style guide on lead-in sentences is to make sure the lead-in is a complete sentence.
```suggestion
Knative Serving includes the following features:
```